src/common/helpers/check-userRole.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from 'src/modules/users/entities/user.entity';
+import { RoleEnum } from '../enums';
+
+export const CheckUserRole = (user: User, role: RoleEnum) => {
+	const userRoles = user.dataValues.roles.map(role => role.name);
+	if (userRoles.includes(role)) {
+		return true;
+	} else {
+		return null;
+	}
```

Comment:
The same effect
return !!user.dataValues.roles.find(userRole => userRole.name === role);

Reply:
You are right, looks like much better.